pkg/services/ngalert/api/prometheus/api_prometheus.go

```diff
@@ -689,9 +689,20 @@ func PrepareRuleGroupStatusesV2(log log.Logger, store ListAlertRulesStoreV2, opt
 	namespaceUIDs := make([]string, 0, len(opts.AllowedNamespaces))
 
 	folderUID := opts.Query.Get("folder_uid")
+	searchFolder := opts.Query.Get("search.folder")
```

Comment:
My HTTP-knowledge isn't comprehensive enough to know if this is fine. Is this fine?